source/conf.py

```diff
@@ -94,6 +100,15 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 
+# adds the actual variables for calling in layout.html to allow for
+# fully configurable sourcecode linking
+html_context = {
+    'github_user' : github_user,
+    'github_repo' : github_repo,
+    'source_suffix' : source_suffix,
+    'github_host' : github_host,
+    'github_branch' : github_branch}
```

Comment:
It would be probably better to move the end bracket to the next line for clearness.